build.gradle

```diff
@@ -16,6 +16,7 @@ description = """Java bindings for libstorj"""
 
 sourceCompatibility = 1.7
 targetCompatibility = 1.7
+def javaHome = System.getenv("JAVA_HOME")
```

Comment:
Please add an empty line above this line.

Reply:
I added and re-pushed.